apps/frontpage/docs-versions.ts

```diff
@@ -1,21 +1,29 @@
+export type DocsVersion = {
```

Comment:
For @kylegach — Confirm this still works after CJS → ESM and location changes.

Reply:
It does not: https://github.com/storybookjs/web/pull/34/files#diff-f81db2672a54c32bb8dea3ef9d967f490626798d37ea4b3c59e1b10ad09dbc15R1